costmap:

    publish_frequency: 2.0
    update_frequency: 2.0

    plugins: 
        - {name: static_map, type: "costmap_2d::StaticLayer"}
        - {name: obstacle_layer, type: "costmap_2d::ObstacleLayer"}
        - {name: inflation_layer,  type: "costmap_2d::InflationLayer"}
    
    # inflation_layer:
        # inflation_radius: 1
        # cost_scaling_factor: 200

    obstacle_layer:
        footprint_clearing_enabled: false
        observation_sources: laser_scan_sensor
        laser_scan_sensor: {
            sensor_frame: robot0_laser_0,
            data_type: LaserScan,
            topic: /robot0/laser_0,
            marking: true,
            clearing: true,
            obstacle_range: 6.0,
            raytrace_range: 6.0
        }

    footprint: [[-0.4, 0.13], [0.4, 0.13], [0.4, -0.13], [-0.4, -0.13]]
    global_frame: /map
    robot_base_frame: robot0

